### PR TITLE
[AS-80] Rotating models

### PR DESCRIPTION
**Pull request by mslup**

### CODE REVIEW

File: src/components/button/Button.tsx [hunk 1]
```diff
@@ -10,21 +10,13 @@ type ButtonProps = {
 
 const Button = ({
   children,
-  onClick = () => {},
+  onClick = () => { },
```
Comment (GitQueenZofia) [inline src/components/button/Button.tsx]: linting

**Review by GitQueenZofia — DISMISSED** [review]
great job!

File: src/components/button/Button.tsx (hunk 1 above)
Comment (mslup) [reply src/components/button/Button.tsx]:
```suggestion
  onClick = () => {},

```

**Comment by mslup** [comment]
already merged in #19

**Pull request closed without merging**